src/tox/config/__init__.py

```diff
@@ -1147,17 +1146,10 @@ def line_of_default_to_zero(section, name=None):
                 known_factors.update(env.split("-"))
 
         # configure testenvs
-        to_do = []
         failures = OrderedDict()
         results = {}
         cur_self = self
```

Comment:
can also probably remove this aliasing too now that it's unused

was this threaded maybe to avoid some subprocess overhead?

Reply:
will tox open any subprocesses during parsing?

Reply:
I believe it does (or used to) to figure out the python versions